data/xml/1982.tc.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1982.tc">
+  <volume id="1" ingest-date="2021-08-04">
+    <meta>
+      <booktitle>Translating and the Computer: Term banks for tomorrow’s world: Translating and the Computer 4</booktitle>
```

Comment:
I would use "Proceedings of Translating and the Computer 4: Term banks for tomorrow's world"